app/static/js/eventKiosk.js

```diff
@@ -51,6 +53,40 @@ function eventFlasher(flash_message, status){
 
 }
 
+$('.qr-reader-button').on("click", function() {
+  //  Opens the camera to scan the ID
+  $('#qr-reader').toggle()
+  let lastResult, countResults = 0;
+  function onScanSuccess(decodedText, decodedResult) {
```

Comment:
I would add a check to make sure we have a full b-number, to avoid submitting that case where it scanned a partial one